### DISCUSSION TITLE
PDCCH simulation

### DISCUSSION THREAD

**Discussion by cbusan123** [NVlabs/sionna#148] [Q&A]
Hi,

I am new to sionna and would like to explore and dig more on PDCCH encoding/decoding part with LLS simulator.

Would like to know does sionna provides API's for PDCCH concepts ? would appreciate if you share more tutorials from sionna LLS simulator perspective.

Thanks in advance.

**Comment by jhoydis** [comment]
Hi, PDCCH is currently not implemented.
What type of tutorial do you find missing?

**Reply by cbusan123** [reply]
Hi, I am looking for PDCCH Blind decoding part which generally does at UE in CONNECTED mode to decode the DCI information.
I would like to take some metrics from existing PDCCH blind decoding mechanisms like power consumption, latency, etc.

I would appreciate if you could provide pointer with sionna simulation.
Thanks in advance.

**Reply by jhoydis** [reply]
As I said above, PDCCH is currently not implemented.

**Reply by cbusan123** [reply]
Ok. Thanks for prompt response.
